ProjectLighthouse.Servers.Website/Pages/UserSettingsPage.cshtml

```diff
@@ -140,8 +141,13 @@ function onSubmit(e){
     let selectedAvatar = "";
     document.getElementById("avatar").addEventListener("change", e => {
       const file = e.target.files.item(0);
+      const img = System.Drawing.Image.FromFile(file);
```

Comment:
This won't work because you're mixing client-side JavaScript with server-side C#. This either needs to be refactored to only use JS code or the image dimension check should be done on the server. Personally, I prefer option 2 because it prevents people from bypassing the client-side check by just crafting their own HTTP request.